### PR TITLE
Remove redundant headers for Managed

### PR DESCRIPTION
**Pull request by tiffwong**
## How to test

Check the Managed section for any remaining redundant headers

### CODE REVIEW

File: packages/manager/src/features/Managed/Contacts/Contacts.tsx [hunk 1]
```diff
@@ -134,143 +132,136 @@ const Contacts: React.FC<CombinedProps> = (props) => {
         the event of a support issue. Create contacts and assign them to a
         group, then assign the group to the appropriate monitor(s).
       </Typography>
-      <div className={classes.root}>
```
Comment (tiffwong) [inline packages/manager/src/features/Managed/Contacts/Contacts.tsx:137]: Only change is removing `<div>` and `classes.root`

**Review by DevDW — APPROVED** [review]
LGTM 👍 

Unrelated to redundant headers, I noticed the "Monitor" column header has a lot of left padding for some reason:

Screen Shot 2021-08-16 at 10 16 21 AM

**Comment by tiffwong** [comment]
@DevDW that's because there's normally an icon that shows up if you have a monitor

Screen Shot 2021-08-16 at 4 20 23 PM

**Review by bnussman — APPROVED** [review]
Looks good!

**Pull request merged**